.travis/install.sh

```diff
@@ -23,9 +23,14 @@ if [ -n "${OPENSSL}" ]; then
         shlib_sed
         make depend
         make -j"$(nproc)"
-        # avoid installing the docs
-        # https://github.com/openssl/openssl/issues/6685#issuecomment-403838728
-        make install_sw install_ssldirs
+        if [[ "${OPENSSL}" =~ "1.0.1" ]]; then
+            # OpenSSL 1.0.1 doesn't support installing without the docs.
```

Comment:
😆 one day we'll get to drop OpenSSL 1.0.1 support.

Reply:
some day